error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/OptionalRules.java

```diff
@@ -158,7 +160,8 @@ Optional<T> after(T input) {
   static final class MapOptionalToBoolean<T> {
     @BeforeTemplate
     boolean before(Optional<T> optional, Function<? super T, Boolean> predicate) {
-      return optional.map(predicate).orElse(Refaster.anyOf(false, Boolean.FALSE));
+      return Refaster.anyOf(
+          optional.map(predicate).orElse(false), optional.map(predicate).orElse(Boolean.FALSE));
```

Comment:
Had to rewrite this as this was causing a "false" positive, `orElse(Refaster.anyOf(false, Boolean.FALSE))` got rewritten to `orElseGet(() -> Refaster.anyOf(false, Boolean.FALSE))`

Reply:
Ah, ideally the new matcher ignores `Refaster.anyOf` calls, since those aren't _true_ method invocations.